docs/usage.md

```diff
@@ -75,6 +75,20 @@ If you are using [GENCODE](https://www.gencodegenes.org/) reference genome files
 * The `--gtf_group_features_type` parameter will automatically be set to `gene_type` as opposed to `gene_biotype`, respectively.
 * If you are running Salmon, the `--gencode` flag will also be passed to the index building step to overcome parsing issues resulting from the transcript IDs in GENCODE fasta files being separated by vertical pipes (`|`) instead of spaces (see [this issue](https://github.com/COMBINE-lab/salmon/issues/15)).
 
+## Adapt pipeline parameters for prokaryotes
+
+The default settings of the pipeline are mainly adapted for eukaryotes but have to be changed slightly for prokaryotes. The main reason for this is the different genetic architecure of prokaryotes. The below mentioned parameters work if a `gff` file is provided as reference. 
+
+Changes and parameter specifications for prokaryotes:
+* `--featurecounts_feature_type transcript` since the default value `exon` does not contain the required `--featurecounts_group_type gene_biotype` specification.
```

Comment:
If you check the gtf file, __exon__ rows don't contain a 'gene_biotype' in the last column. Only the __transcript__ rows have this information. My suggestion is as you mention above *--featurecounts_feature_type "transcript" --featurecounts_group_type "gene_biotype"*. Since the second part (...group_type) is already the default setting you don't need to include it in the command.

Reply:
I see, you are right. Could it be that this varies? I thought I did use recently (with the old rnaseq version) "transcript" but there was still no "gene_biotype".

My point here is that "exon" is never a good choice for bacterial RNA-Seq. In the files you linked, "exon" is present 121 times but "transcript" 4417 times in the gtf file. In the source gff file its 121 "exon" but 4298 CDS (which might be also an option) and 4359 "gene".